packages/react-components/react-avatar/src/components/Avatar/useAvatarStyles.ts

```diff
@@ -149,28 +136,12 @@ const useStyles = makeStyles({
   inactive: {
     opacity: '0.8',
     transform: 'scale(0.875)',
-
-    transitionProperty: 'transform, opacity',
-    transitionDuration: `${tokens.durationUltraSlow}, ${tokens.durationFaster}`,
```

Comment:
`transitionProperty` and `transitionDuration` are the same here as in the `activeOrInactive` class and don't need to be repeated.